raiden/tests/integration/long_running/flaky/test_token_networks.py

```diff
@@ -5,11 +5,54 @@
 
 from raiden.api.python import RaidenAPI
 from raiden.tests.utils.geth import wait_until_block
-from raiden.transfer.state import CHANNEL_STATE_SETTLED
+from raiden.transfer import views
+from raiden.transfer.state import NODE_NETWORK_REACHABLE
+from raiden import routing
 
 log = structlog.get_logger(__name__)
 
 
+def wait_for_reachable_node(sender, receiver):
+    """Wait until a node is reachable"""
+    while True:
+        state = RaidenAPI(sender).get_node_network_state(
+            receiver.address,
+        )
+        if state == NODE_NETWORK_REACHABLE:
+            return
+        gevent.sleep(1)
+
+
+def wait_for_transaction(
```

Comment:
same thing as the above function. Check if any of the functions in `waiting.py` fulfill your needs instead. If not please move this also there with a configurable `retry_timeout`